File: Kattis/Jug_Hard.py
def gcd(a,b):
    while b:
        a,b = b, a%b
    return a
for _ in xrange(input()):
    print ['No', 'Yes'][(lambda x: x[2]%gcd(x[0],x[1]) == 0)(map(int, raw_input().split()))]

File: Kattis/Jug_Hard.hs
import Control.Monad
slve [a,b,c] = if mod c (gcd a b) == 0 then "Yes" else "No"
readInts :: String -> [Int]
readInts = map read . words
main = do
    t <- readLn :: IO Int
    forM_ [1..t] (\_ -> do
    	l <- fmap (slve. readInts) getLine
    	putStrLn l)
